configure.php

```diff
@@ -198,13 +234,13 @@ function delete_files( string|array $paths ) {
 
 $author_name = ask(
 	question: 'Author name?',
-	default: run( 'git config user.name' ),
+	default: run( 'git config user.name' ) ?: ( $args['author_name'] ?? '' ),
```

Comment:
I'd suggest having the args take priority over the `git config` values. As a user, I don't have to provide args, but if I do, I'd expect them to take precedence.

Reply:
That is true. Though this would make it so that _any_ plugin scaffolded with the scaffolder will always have this default. Guess it's a fair tradeoff since most folks won't have `git` setup with a username/email in Broadway.

Reply:
Agree. For Alley's purposes, this is consistent with the starter plugin's authorship, yet it can be overridden if desired.